gpii/node_modules/contextManager/src/ContextManager.js

```diff
@@ -96,11 +100,13 @@ fluid.defaults("gpii.contextManager", {
  * Responsible for weeding out any information from the request body that is not strictly
  * context variables and then notifying the applier.
  *
- * @that {Object} The context manager component
- * @body {Object} the body of the request. It is expected to contain only key-value pairs
+ * @param that {Component} The context manager component
+ * @param body {Object} the body of the request. It is expected to contain only key-value pairs
  *      of context variables, and optionally a 'timestamp' key (and associated value) which
  *      will not be considered part of the current context
  **/
+// TODO: Why do we confusingly have this thing named "context" which is always just a copy of the environment
+// with the timestamp removed?
```

Comment:
Not sure which "context" you're referring to, but the currentContext should always hold the current environment.. It is used in the modelListener (and on login) for checking what the current "activeContext" should be.

I would say that we do need some place to store the current environmental context, but other than that there might be a better way to handle it... Perhaps to avoid confusion we should call the model variable "currentEnvironment" instead of "currentContext"?

Reply:
OK, done. Still not sure why the surplus timestamp is in the body, but that is a tale for another day...
:mouse: